server/requirements.txt

```diff
@@ -35,4 +35,6 @@ yapf==0.31.0
 geojson_rewind==1.0.1
 markupsafe==2.0.1
 python-dateutil==2.8.2
-pandas>=1.3.5
\ No newline at end of file
+pandas>=1.3.5
+spacy>=3.0.0,<4.0.0
+en_core_web_lg @ https://github.com/explosion/spacy-models/releases/download/en_core_web_lg-3.4.1/en_core_web_lg-3.4.1-py3-none-any.whl
```

Comment:
from my understanding what does it do?  (and if comments work in requirements.txt considering adding one)

Reply:
Added some comments. Found this approach recommended on their website. It installs the language model for NER as a package.